Lib/test/test_enum.py

```diff
@@ -15,13 +16,14 @@
 from test.support import threading_helper
 from datetime import timedelta
 
-def load_tests(loader, tests, ignore):
-    tests.addTests(doctest.DocTestSuite(enum))
-    tests.addTests(doctest.DocFileSuite(
-            '../../Doc/library/enum.rst',
-            optionflags=doctest.ELLIPSIS|doctest.NORMALIZE_WHITESPACE,
-            ))
-    return tests
+if os.path.exists('../../Doc/library/enum.rst'):
```

Comment:
Would be possible to start from test.support.TEST_HOME_DIR to not rely on the current directory? Something like:

```
ENUM_RST = os.path.normpath(os.path.join(TEST_HOME_DIR, '..', '..', 'Doc', 'library', enum.rst'))
if os.path.exists(ENUM_RST): ...
```

Can tests.addTests(doctest.DocTestSuite(enum)) be run even if enum.rst is not available?

Reply:
Using `TEST_HOME_DIR` results in `ValueError: Module-relative files may not have absolute paths`.

Reply:
Oh, ok!